input/pagecontent/reference.md

```diff
@@ -1682,13 +1700,23 @@ Creating a code system uses the keywords `CodeSystem`, `Id`, `Title` and `Descri
 * #{code} "{display string}" "{definition string}"
 ```
 
+Codes that are contained within other codes can also be defined. The result is a hierarchical structure of codes within a code system. To define such codes, list all of the preceding codes in the hierarchy before the new code:
+
+
+```
+* #{existing code} #{code} "{display string}" "{definition string}"
+```
+
 **Notes:**
 
 * There MUST NOT be a code system before the hash sign `#`. The code system name is given by the `CodeSystem` keyword.
 * The definition of the term, provided as the second string following the code, is RECOMMENDED.
 * Do not use the word `include` in a code system rule. The rule is creating a brand new code, not including an existing code defined elsewhere.
+* When defining hierarchical codes, a code must be defined before it can be used to specify a hierarchy.
```

Comment:
I think the phrase "can be used to specify a hierarchy" might be unclear.  Consider:
```suggestion
* When defining hierarchical codes, parent codes must be defined before their children.
```

Reply:
Perhaps we should also include a note that caret syntax can be used to define other metadata attributes for each code (e.g., designation)?